src/main/java/org/folio/circulation/domain/notice/schedule/ScheduledNoticeHandler.java

```diff
@@ -201,16 +201,12 @@ protected boolean noticeShouldNotBeSent(ScheduledNoticeContext context) {
     if (result.succeeded()) {
       log.info("Finished processing scheduled notice {}", notice.getId());
       return completedFuture(result);
-    }
-
-    HttpFailure failure = result.cause();
-    log.error("Processing scheduled notice {} failed: {}", notice.getId(), failure);
+    } else {
```

Comment:
No need for `else`, there is a `return` in `if` above

Reply:
Else removed